docs/guides/Screen-Navigation-Prop.md

```diff
@@ -116,6 +116,26 @@ class HomeScreen extends React.Component {
 }
 ```
 
+*Going Back from a Specific Screen*
+
+Consider the following navigation stack history:
+```... 
+navigation.navigate(SCREEN_KEY_A);
+...
+navigation.navigate(SCREEN_KEY_B);
+...
+navigation.navigate(SCREEN_KEY_C);
+...
+navigation.navigate(SCREEN_KEY_D);
+```
+
+Now you are on *Screen D* and want to go back to *screen A* (popping D, C, and B).
```

Comment:
Either capitalize screen in both cases, or in neither (`Screen D` vs `screen A`)